docs/sources/shared/upgrade/upgrade-common-tasks.md

```diff
@@ -8,7 +8,9 @@ Because Grafana upgrades are backward compatible, the upgrade process is straigh
 In addition to common tasks you should complete for all versions of Grafana, there might be additional upgrade tasks to complete for a version.
 
 {{% admonition type="note" %}}
-There might be minor breaking changes in some releases. We outline these changes in the [What's New ]({{< relref "../../whatsnew/" >}}) document for each release.
+
+> **Note:** There might be minor breaking changes in some releases. We outline these changes in the [What's New ]({{< relref "../../whatsnew/" >}}) document for most releases or the [Breaking changes]({{< relref "../../breaking-changes/" >}}), in some cases.
```

Comment:
```suggestion
> **Note:** There might be breaking changes in some releases. We outline these changes in the [What's New ]({{< relref "../../whatsnew/" >}}) document for most releases or a separate [Breaking changes]({{< relref "../../breaking-changes/" >}}) document for releases with many breaking changes.
```

Reply:
Good call. I'll make this change on my local and push it since I've got some edits going.